tracetesting/funcs.bash

```diff
@@ -26,7 +26,11 @@ test() {
     echo -n "OK"
   else
     echo "FAIL"
+    echo "$name.json:"
     cat results/responses/$name.json
+    echo "$name.xml:"
+    cat results/$name.xml
```

Comment:
Wanted to debug a problem with the e2e tests and figured this file could help with that.